packages/breadcrumbs/CHANGELOG.md

```diff
@@ -15,6 +15,7 @@
 
 ## Versions
 
+* [v2.1.2 - New version of link-list](#v212)
```

Comment:
A major breaking change in a dependency that dictates the HTML should probably also be a major breaking change for the parent package... no?

Reply:
I assumed because it didn't use that feature ( `children` ) that it wouldn't be required in this component. Though you are correct if someone was using `children` in breadcrumbs it would break currently.